extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -218,3 +238,69 @@ function format_track_duration( $duration ) {
 
 	return date_i18n( $format, $duration );
 }
+
+/**
+ * Build an array with CSS classes and inline styles defining the colors
+ * which will be applied to player either in the editor-canvas
+ * as well as in the front-end side.
+ *
+ * @param  array $attributes block attributes.
+ * @return array Colors CSS classes and inline styles.
+ */
+function get_colors( $attributes ) {
+	$colors = array(
+		'css_classes'      => array(),
+		'css_text_classes' => array(),
+		'style'            => '',
+		'class_name'       => '',
+		'text_class_name'  => '',
+	);
```

Comment:
I'm a bit unclear about all these return values.

- `class_name` is never used.
- `text_class_name` might be unnecessary. We already return `css_text_classes`, we might as well `implode` it outside of the `get_colors` function.
- What about updating `css_classes` for the background colors only, and then merge it with `css_text_classes` at the end? Something like this:
```php
if ( $has_named_text_color ) {
  $colors['css_text_classes'][] = 'has-text-color';
}
// ...

$colors['css_classes'] = array_merge( $colors['css_classes'], $colors['css_text_classes'] );

return $colors;
```